app/helpers/herbaria_helper.rb

```diff
@@ -3,11 +3,9 @@
 # helper methods for Herbaria views
 module HerbariaHelper
   def herbarium_top_users(herbarium)
-    Herbarium.connection.select_rows(%(
-      SELECT u.name, u.login, COUNT(u.id)
-      FROM herbarium_records hr JOIN users u ON u.id = hr.user_id
-      WHERE hr.herbarium_id = #{herbarium.id}
-      GROUP BY u.id ORDER BY COUNT(u.id) DESC LIMIT 5
-    ))
+    User.joins(:herbarium_records).
+      where(HerbariumRecord[:id].eq(herbarium.id)).
```

Comment:
Yep, this line is off.  It might be worth checking to see why no tests catch this.  This should totally be unit-testable.

Reply:
Good catch, thanks. Added tests